packages/harmony/src/components/button/types.ts

```diff
@@ -155,7 +155,8 @@ export enum FilterButtonType {
 }
 
 export type FilterButtonOption = {
-  label: string
+  value: string
```

Comment:
Same question here re: changing the interface to the component and whether that's going to break any existing usage.

Reply:
luckily typescript catches these - we only had a storybook and our own usage.